BuildAll.ps1

```diff
@@ -154,12 +165,17 @@ Try {
         #------------------
 
         #Restore packages from mrt.
-        & .\.nuget\nuget.exe restore "$MRTSourcesDirectory\mrt\MrtCore.sln" -ConfigFile nuget.config
+        & .\.nuget\nuget.exe restore "$MRTSourcesDirectory\mrt\MrtCore.sln" -ConfigFile nuget.config -ConfigFile nuget.config -PackagesDirectory "$MRTSourcesDirectory\mrt\packages"
```

Comment:
It is unclear why "-ConfigFile nuget.config" is duplicated.

Reply:
Whoops typo